script.js

```diff
@@ -0,0 +1,20 @@
+const hamburg = document.querySelector('.menu_for_mbl');
+const overlay = document.querySelector('.open');
+const mainMenu = document.querySelector('.main-menu');
+const cancel = document.querySelector('.cancel');
+
+const port = document.querySelector('.portfolio');
+
+hamburg.addEventListener('click', () => {
+  mainMenu.style.display = 'flex';
+  overlay.style.display = 'block';
+});
+
+cancel.addEventListener('click', () => {
+  overlay.style.display = 'none';
+});
```

Comment:
Please make sure that when you click on any option in your hamburger menu the user is redirected to that specific section of your portfolio and the hamburger menu closes. This feature currently is working just for the contact option. :+1: :smiley: :smiley: